Source/WebKit/Shared/AuxiliaryProcessExtensions/com.apple.WebKit.appexpt

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>com.apple.webkit.webcontent.extension</key>
+    <dict>
+        <key>EXExtensionPointIsPublic</key>
+        <true/>
```

Comment:
I think you want all these `EXExtensionPointIsPublic` values to be `<false/>`. These are not extension points that third-party apps can implement extensions for.

Reply:
Fixed!

Thanks for reviewing, weinig, xeenon, and mew-apple!